.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -16,6 +13,46 @@ on:
         required: true
         default: true
         type: boolean
+      owner-subject:
+        description: 'Owner (used for creating tmp billing project)'
+        required: true
+        default: 'hermione.owner@quality.firecloud.org'
+        type: string
+      non-owner-subject:
+        description: 'Student (non-owner used for e2e testing)'
+        required: true
+        default: 'harry.potter@quality.firecloud.org'
+        type: string
+      service-account:
+        description: 'Email address or unique identifier of the Google Cloud service account for which to generate credentials'
+        required: true
+        default: 'firecloud-qa@broad-dsde-qa.iam.gserviceaccount.com'
+        type: string
+      access-token-scopes:
+        description: 'List of OAuth 2.0 access scopes to be included in the generated token'
+        required: true
+        default: 'profile, email, openid'
+        type: string
+      tenant-id:
+        description: 'Azure tenant ID. The default tenant is DSP Terra Dev'
+        required: true
+        default: 'fad90753-2022-4456-9b0a-c7e5b934e408'
+        type: string
+      subscription-id:
+        description: 'Azure subscription ID'
+        required: true
+        default: 'f557c728-871d-408c-a28b-eb6b2141a087. The default subscription is 8201558-dsp-azure-testing'
+        type: string
+      mrg-id:
+        description: 'Azure Managed Resource Group name. The default is staticTestingMrg within subscription 8201558-dsp-azure-testing.'
+        required: true
+        default: 'staticTestingMrg'
+        type: string
+      landing-zone-id:
+        description: 'Landing Zone ID. An existing LZID tag within a given MRG.'
+        required: true
+        default: 'f41c1a97-179b-4a18-9615-5214d79ba600'
```

Comment:
Note that the landing zone will need to be recreated from time to time, which means that all the places that reference the ID will need to be updated. So it would be good to think of how to centralize this so we don't have to go around and change a lot of workflow files.

Reply:
Agreed. Maybe we should have a wilder discussion about a centralized process to enable these IDs to be ingested from the maintainers of these landing zone resources?